techniques/general/G131.html

```diff
@@ -24,16 +24,15 @@ <h3>A form with required fields</h3>
       </section>
    </section><section id="tests"><h2>Tests</h2>
       <section class="procedure"><h3>Procedure</h3>
-         <p>For each interface component in the content:</p>
```

Comment:
While this change makes sense for 2.4.6 this technique applies to SC 3.3.2 as well and this technique 131 is the foundation for other compound techniques (see the AND clause in understanding SC 3.3.2) and thus I think it is necessary for us to not assume a label is present and that the check needs to test to make sure a label is present.  So I don't think we should change it.

Reply:
@mraccess77 there is a note specifically about this already existing in the technique: "Note: This technique must be combined with other techniques to meet SC 3.3.2. See Understanding SC 3.3.2 for details."

G131 is about making existing labels clear, not about requiring a label to be present for every UI component. Confusion over the existing wording in the procedure is what initiated the issue. Remember that G131 can apply to things besides inputs -- things that are not required to have labels according to WCAG. Where a label in on an input, it is going to be covered by 3.3.2.